src/autocomplete.js

```diff
@@ -534,6 +534,9 @@ yourlabs.Autocomplete.prototype.hide = function() {
 // This function is in charge of hilighting the right result from keyboard
 // navigation.
 yourlabs.Autocomplete.prototype.move = function(e) {
+	if (this.value === null)
```

Comment:
Could you fix indentation for this line please ?

We use 4 spaces for indentation.